decisions/wc_society_decisions.txt

```diff
@@ -288,35 +288,77 @@ society_decisions = {
 		only_playable = yes
 
 		potential = {
-			is_adult = yes
-			is_incapable = no
-			prisoner = no
+			is_landed = yes
+			
 			society_member_of = cenarion_circle
 			society_rank > 3
-			capital_scope = {
-				NOT = { hospital_has_building = wc_cenarion_enclave }
+			trigger_if = {
+				limit = { has_dlc = "Reapers" }
+				capital_scope = { NOT = { hospital_has_building = hp_cenarion_enclave } }
+			}
+			trigger_else = {
+				capital_scope = {
+					NOT = {
+						any_province_holding = {
+							has_building = tp_cenarion_enclave
+						}
+					}
+					trigger_if = {
+						limit = { has_empty_holding = yes }
+						always = yes
+					}
+					trigger_else = {
+						any_province_holding = {
+							holding_type = temple
+							holder_scope = {
+								OR = {
+									character = ROOT
+									is_liege_or_above = ROOT
+								}
+							}
+						}
+					}
+				}
 			}
 		}
 
 		allow = {
-			is_inaccessible_trigger = no
 			society_currency >= 1500
 		}
 
 		effect = {
 			change_society_currency = -1500
 			capital_scope = {
 				if = {
-					limit = {
-					has_hospital = no
-					}
-				  	build_holding = {
-						type = hospital
-						holder = ROOT
+					limit = { has_dlc = "Reapers" }
+					if = {
+						limit = { has_hospital = no }
+						build_holding = {
+							type = hospital
+							holder = ROOT
+						}
 					}
+					add_to_extra_holding = { type = hospital building = hp_cenarion_enclave }
 				}
-				add_to_extra_holding = { type = hospital building = wc_cenarion_enclave }
+				else = {
+					if = {
+						limit = { has_empty_holding = yes }
+						build_holding = { type = temple holder = ROOT }
+					}
+					random_province_holding = {
+						limit = {
+							holding_type = temple
+							holder_scope = {
+								OR = {
+									character = ROOT
+									is_liege_or_above = ROOT
+								}
+							}
+						}
+						add_building = tp_cenarion_enclave
+					}
```

Comment:
> Build temple and gain Cenarion Enclave in random holding

This should be clarified in power description or changed, since player may think that special temple building will be included in newly created temple or that you will only gain temple if you already don't have one.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/74903401-da9d0480-53b9-11ea-994e-864faffecace.png)
![image](https://user-images.githubusercontent.com/46576860/74903022-cc021d80-53b8-11ea-986e-406bc94065bf.png)

</details>